client/packages/invoices/src/InboundShipment/ListView/AppBarButtons.tsx

```diff
@@ -78,7 +78,7 @@ export const AppBarButtons: FC<{
                 .addPart(AppRoute.InboundShipment)
                 .addPart(String(invoiceNumber))
                 .build(),
-              { replace: true }
+              { replace: false }
```

Comment:
Can I confirm the expected behaviour for the requisition detail ones. I'm thinking of this in terms of breadcrumbs back, rather than back being 'whichever view it was on before'

- Requisition Detail -> add item/click item row (input page opens) -> back button will go back to Requisition Detail. Back once more is to Requisition List

- Requisition Detail -> add item/click item row (input page opens) -> close button takes you back to Requisition Detail.
-> if now clicking the back button I would expect it to take me to the Requisition List (not back to the input page)
Similar thing for Next/Previous, these should go back to the the Detail view, and then back to List view?